.github/workflows/reusable-algolia-sync.yml

```diff
@@ -130,6 +130,21 @@ jobs:
           SQUIDEX_BASE_URL: ${{ steps.setup.outputs.squidex-base-url }}
           SQUIDEX_CLIENT_ID: ${{ steps.setup.outputs.squidex-client-id }}
           SQUIDEX_CLIENT_SECRET: ${{ secrets.SQUIDEX_CLIENT_SECRET }}
+      - name: Import Events (EVENTS | ALL)
+        if: ${{ inputs.entity == 'events' || inputs.entity == 'all'}}
```

Comment:
do we want to make it reusable action?

Reply:
https://github.com/yldio/asap-hub/runs/6709108421?check_suite_focus=true